enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -1517,12 +1546,11 @@ func (c *FirecrackerContainer) getJailerConfig(kernelImagePath string) (*fcclien
 		Stdout:         c.vmLogWriter(),
 		Stderr:         c.vmLogWriter(),
 		CgroupVersion:  cgroupVersion,
-		CgroupArgs:     cgroupArgs,
 		// Use the root cgroup as the cgroup parent rather than the default
 		// "firecracker" subdir, so that VM cgroups are siblings of OCI
 		// container cgroups. This is needed because CPU weight is applied to
 		// sibling cgroups in a hierarchy.
-		ParentCgroup: fcclient.String(""),
+		ParentCgroup: fcclient.String(c.cgroupParent),
```

Comment:
I think you want to use c.cgroupPath() here. Otherwise that cgroup is never used.

Reply:
updated the comment - to get the effective cgroup path, the jailer prepends /sys/fs/cgroup and appends the VM ID.